src/test/Makefile

```diff
@@ -711,7 +711,7 @@ $(OBJECT_DIR)/gtest_main.a : $(OBJECT_DIR)/gtest-all.o $(OBJECT_DIR)/gtest_main.
 
 # includes in test dir must override includes in user dir, unless the user
 # specifies a list of endorsed directories in ${target}_INCLUDE_DIRS.
-test_include_dirs  = $1 $(TEST_DIR) $(USER_DIR)
+test_include_dirs  = $1 $(TEST_DIR) $(USER_DIR) $(ROOT)/lib/main/MAVLink
```

Comment:
If possible, I'd prefer to remove `main/MAVLink` from main compilation includes.
`#include "common/mavlink.h"` is extremely confusing. `#include "MAVLink/common/mavlink.h" is IMO better

Reply:
@demvlad have you tried ?